gui/public/i18n/en/translation.ftl

```diff
@@ -283,6 +283,13 @@ settings-general-steamvr-trackers-feet = Feet
 settings-general-steamvr-trackers-knees = Knees
 settings-general-steamvr-trackers-elbows = Elbows
 settings-general-steamvr-trackers-hands = Hands
+settings-general-steamvr-trackers-tracker_toggling = Automatic tracker switching
+settings-general-steamvr-trackers-tracker_toggling-description = Automatically handles the turning on and off of SteamVR trackers depending on your current body assignment of trackers
+settings-general-steamvr-trackers-tracker_toggling-label = Automatic tracker switching
+settings-general-steamvr-trackers-hands-warning = <b>Warning:</b> Hands trackers will replace the tracking
+    of your controllers, are you sure?
```

Comment:
```suggestion
settings-general-steamvr-trackers-tracker_toggling = Automatic tracker assignment
settings-general-steamvr-trackers-tracker_toggling-description = Automatically handles toggling SteamVR trackers on or off depending on your current tracker body assignments
settings-general-steamvr-trackers-tracker_toggling-label = Automatic tracker assignment
settings-general-steamvr-trackers-hands-warning = <b>Warning:</b> hands trackers will override your controllers. Are you sure?
```

Not too sure about "current tracker body assignments", but it sounds better than "current body assignment of trackers" to me.

Reply:
seems good, the line break is in purpose as the warning becomes *loong* otherwise

Reply:
Added a line break before “Are you sure?”. How’s that?

Reply:
```suggestion
settings-general-steamvr-trackers-tracker_toggling = Automatic tracker assignment
settings-general-steamvr-trackers-tracker_toggling-description = Automatically handles toggling SteamVR trackers on or off depending on your current tracker assignments
settings-general-steamvr-trackers-tracker_toggling-label = Automatic tracker assignment
settings-general-steamvr-trackers-hands-warning = <b>Warning:</b> hands trackers will override your controllers.
    Are you sure?
```
this fine?